lua/lazyvim/plugins/coding.lua

```diff
@@ -54,9 +54,13 @@ return {
             if icons[item.kind] then
               item.kind = icons[item.kind] .. item.kind
             end
-            if entry.context.filetype == "rust" then
-              item.menu = nil
-            end
+
+            local abbr_width = vim.g.cmp_widths and vim.g.cmp_widths.abbr or 30
+            local menu_width = vim.g.cmp_widths and vim.g.cmp_widths.menu or 30
+
+            item.abbr = vim.fn.strcharpart(item.abbr, 0, abbr_width - 1) .. "…"
```

Comment:
so someting like `if vim.fn.strdisplaywidth(item.abbr) > abbr_width then`....

Reply:
Oh, now I understand what you mean. Just a sec.

Reply:
I also wrapped the trimming inside an `if item.menu` because on LSPs that didn't have `item.menu` it showed another thing in place of the non-existent `item.menu` (like Lua for example).